neurons/validators/scraper_validator.py

```diff
@@ -450,8 +457,18 @@ def log_event(self, tasks, event, start_time, uids, rewards):
 
         bt.logging.debug("Run Task event:", event)
 
-    async def query_and_score(self, strategy=QUERY_MINERS.RANDOM):
+    async def query_and_score(self, strategy=QUERY_MINERS.RANDOM, model: Optional[Model] = None):
         try:
+            # Default model if none provided
+            if model is None:
+                model = Model.NOVA
+
+
+            #lazy import that function should be in another .py file
+            from neurons.validators.api import get_max_execution_time
```

Comment:
we can move get_max_execution_time to utils.py if we need in both scraper validator and api and avoid circular import